core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -140,6 +147,59 @@ export class AdminBackendApiService {
       });
     });
   }
+
+  private _postAdminActionAsync(action:string, payload:Object): Promise<void> {
+    return this.http.post<void>(
+      AdminPageConstants.ADMIN_HANDLER_URL, { action, ...payload }).toPromise();
+  }
+
+  async startNewJobAsync(jobType: string): Promise<void> {
+    let action = 'start_new_job';
+    let payload = {
+      job_type: jobType
+    };
+    return this._postAdminActionAsync(action, payload);
+  }
+
+  async cancelJobAsync(jobId: string, jobType: string): Promise<void> {
+    let action = 'cancel_job';
+    let payload = {
+      job_id: jobId,
+      job_type: jobType
+    };
+    return this._postAdminActionAsync(action, payload);
+  }
+
+  async startComputationAsync(computationType: string): Promise<void> {
+    let action = 'start_computation';
+    let payload = {
+      computation_type: computationType
+    };
+    return this._postAdminActionAsync(action, payload);
+  }
+
+  async stopComputationAsync(computationType: string): Promise<void> {
+    let action = 'stop_computation';
+    let payload = {
+      computation_type: computationType
+    };
+    return this._postAdminActionAsync(action, payload);
+  }
+
+  async showJobOutputAsync(jobId: string): Promise<string> {
```

Comment:
fetchJobOutputAsync* (This method is handling any show/hide functionality.)

Reply:
done